x-pack/plugins/enterprise_search/public/applications/applications/components/engines/create_engine_flyout.tsx

```diff
@@ -129,10 +129,10 @@ export const CreateEngineFlyout = ({ onClose }: CreateEngineFlyoutProps) => {
       <EuiFlyoutBody>
         <EuiFlexGroup direction="column">
           <EuiFlexItem grow>
-            <EuiCallOut title="Technical Preview feature" color="warning" iconType="beaker">
+            <EuiCallOut title="Beta feature" color="warning" iconType="beaker">
               <FormattedMessage
                 id="xpack.enterpriseSearch.content.engines.createEngine.technicalPreviewCallOut.title"
```

Comment:
Should we rename this? `xpack.enterpriseSearch.content.engines.createEngine.betaCallOut.title`

Reply:
Yeah. Missed it. I think it's better to generalize translation. So we don't have change every time. 
Something like: 
`xpack.enterpriseSearch.content.engines.createEngine.featureCallOut.title`

Reply:
Add in a3320f6df11786f6342c211c9338214eb9bffb5d